lib/app/components/nuxt-child.js

```diff
@@ -23,14 +23,15 @@ export default {
         transitionProps[key] = transition[key]
       }
     })
+
     let listeners = {}
```

Comment:
`transitionProps` and `listeners` can be const.

Reply:
Oh yes! Will fix that :relaxed: